certified-connectors/EXPOCAD/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e30613",
+    "capabilities": [],
+    "publisher": "Michael Ochs"
+  }
```

Comment:
@mikefactorial Please add `stackOwner` property as well

Reply:
Hi @sriyen-msft this has been resolved in the latest commit